spec/features/users_spec.rb

```diff
@@ -0,0 +1,85 @@
+# rubocop:disable Metrics/BlockLength
+require 'rails_helper'
+
+RSpec.feature 'Users', type: :feature do
```

Comment:
- [ ] **[OPTIONAL]** You added the user index and show page tests in this one file. I think it would have been best to separate them into different files for better code organization.